lib/puppet/transaction/report.rb

```diff
@@ -199,7 +199,7 @@ def compute_status(resource_metrics, change_metric)
   end
 
   # @api private
-  def has_noop_events?(resource)
+  def noop_events?(resource)
```

Comment:
This makes me a little nervous because the `api` tag doesn't prevent someone from accessing it and it's been this way for so long.